testsuite/MDAnalysisTests/coordinates/test_h5md.py

```diff
@@ -0,0 +1,224 @@
+import pytest
+from numpy.testing import assert_almost_equal, assert_array_equal
+import numpy as np
+import MDAnalysis as mda
+from MDAnalysis.coordinates.H5MD import HAS_H5PY
+from MDAnalysisTests.datafiles import H5MD_xvf, TPR_xvf, COORDINATES_TOPOLOGY, COORDINATES_H5MD
+from MDAnalysisTests.coordinates.base import (MultiframeReaderTest,
+                                              BaseReference, BaseWriterTest,
+                                              assert_timestep_almost_equal)
+
+
+h5py = pytest.importorskip('h5py')
```

Comment:
Add 2 blank lines.

With this line, tests will skip without h5py – which is good. Just double check that we are actually testing on Windows.